### PR TITLE
PISO_Code

### PR DESCRIPTION
**Pull request by markshake**

### CODE REVIEW

**Comment by markshake** [comment]
Change the program extension ".F90" to ".f90"

File: code/cavity2d/CAVITY_2D_STAGGERED.f90 [hunk 1]
```diff
@@ -1,507 +1,507 @@
...
-	  ERR=ERR+UTEMP*UTEMP
-	  ENDDO; ENDDO
-	  ERR=SQRT(ERR/(MI*MJ))
-
-
-	  IF(ERR .GT. 0.000001 .OR. ITT .LT. 10) GOTO 1000
-!	  WRITE(*,*)'GS_ITER',ITT,ERR
-      RETURN
-	  END
+!==========================2D CAVITY FLOW=========================================
```
Comment (yungyuc) [inline code/cavity2d/CAVITY_2D_STAGGERED.f90:1]: You are changing this back to dos format again.  Please don’t do this.  You need to configure your editor to do the right thing.

```
$ file CAVITY_2D_STAGGERED.f90
CAVITY_2D_STAGGERED.f90: ASCII text, with CRLF line terminators
```

File: code/cavity2d/CAVITY_2D_STAGGERED.f90 [hunk 2]
```diff
@@ -1,507 +1,507 @@
...
-
-	  IF(ERR .GT. 0.000001 .OR. ITT .LT. 10) GOTO 1000
-!	  WRITE(*,*)'GS_ITER',ITT,ERR
-      RETURN
-	  END
+!==========================2D CAVITY FLOW=========================================  
+! PISO SCHEME  FOR 2D (IN)COMPRESSIBLE FLOW
+! Y. JIANG, CP cHEN, AND HM SHANG: A NEW PRESSURE-VELOCITY COUPLING PROCEDURE FOR
+!       INVICID AND VISCOUS FLOWS AT ALL SPEED
+!
```
Comment (yungyuc) [inline code/cavity2d/CAVITY_2D_STAGGERED.f90:5]: I’ve cleaned up the trailing white spaces for you, and now you add it back.  Please don’t do this.  Be keen to your code quality.

File: code/cavity2d/Makefile [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Default Target
+.PHONY: all
+all: CAVITY_2D_STAGGERED
+
+# CAVITY_2D_STAGGERED
+CAVITY_2D_STAGGERED: clean
+	gfortran -o CAVITY_2D_STAGGERED CAVITY_2D_STAGGERED.f90
```
Comment (yungyuc) [inline code/cavity2d/Makefile:7]: No optimization flag is specified.  Why?

File: code/cavity2d/Makefile [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Default Target
+.PHONY: all
+all: CAVITY_2D_STAGGERED
+
+# CAVITY_2D_STAGGERED
```
Comment (yungyuc) [inline code/cavity2d/Makefile:5]: This comment is unnecessary.  The next line is crystal clear for the target.  Please remove this line.

**Review by yungyuc — CHANGES_REQUESTED** [review]
It’s a big no to revert to dos format and readd all the tailing white spaces.  Please change.

**Comment by yungyuc** [comment]
A side note.  Next time you make PR please give it a meaningful title.  Path 2788 won’t be too distinguishable than patch 2988.

File: code/cavity2d/Makefile (hunk 3 above)
Comment (markshake) [reply code/cavity2d/Makefile:7]: change to gfortran -O3 CAVITY_2D_STAGGERED.f90

File: code/cavity2d/Makefile (hunk 4 above)
Comment (markshake) [reply code/cavity2d/Makefile:5]: # Default Target

CAVITY_2D_STAGGERED: clean
	gfortran -O3 CAVITY_2D_STAGGERED.f90

.PHONY: clean
clean:
	rm -f CAVITY_2D_STAGGERED

File: code/cavity2d/Makefile (hunk 3 above)
Comment (yungyuc) [reply code/cavity2d/Makefile:7]: I am not seeing update?

File: code/cavity2d/Makefile (hunk 4 above)
Comment (yungyuc) [reply code/cavity2d/Makefile:5]: I am not sure I understand your comment?

**Pull request closed without merging**